test/unit/deletion_test.rb

```diff
@@ -61,7 +61,7 @@ class DeletionTest < ActiveSupport::TestCase
 
   teardown do
     # This is necessary due to after_commit not cleaning up for us
```

Comment:
This is only necessary because we `self.use_transactional_fixtures = false` . However on rails 4.2+ we dont need to disable txn fixtures to test after_commits. So we probably could remove that line and this `teardown` method.